src/typechecker/tc-env.lisp

```diff
@@ -4,6 +4,7 @@
    #:coalton-impl/typechecker/base
    #:coalton-impl/typechecker/parse-type)
   (:local-nicknames
+   (#:a #:alexandria)
```

Comment:
Don't PLN alexandria.

Reply:
i'll accept the PLN

    (#:the-library-of-alexandria #:alexandria)

however. send tweet

Reply:
As a compromise, it could be referred to via the Mongolian vowel separator https://github.com/jbouwman/coalton/pull/2